DFC.ServiceTaxonomy.Editor/wwwroot/ncs.js

```diff
@@ -0,0 +1,10 @@
+﻿$(function () {
+    var ul = $("ul#adminMenu li#new ul");
```

Comment:
Could this be simplified to `#new ul`?

Reply:
It could, but I like to be specific in case there are other elements with an ID of "new" - doesn't seem overly unique! What do you think?

Reply:
Id's must be unique within a html document.
The selector is not performance critical, so it doesn't matter too much, its just a little bit tidier.

Reply:
Done!